specifications/config.md

```diff
@@ -0,0 +1,140 @@
+# Configuration
+
+This document specifies how Stormpath SDKs should handle configuration.
+
+## Processing
+
+The order in which the configuration should be processed.
+
+1. Load configuration according to *loading order*.
+2. Check and apply any *post-processing keys*.
+3. Validate the configuration according to the *validation rules*.
+
+## Loading order
+
+The order in which the configuration should be loaded.
+
+1. Default configuration.
+2. `apiKey.properties` file from `~/.stormpath` directory**.
+3. `stormpath.[json or yaml]` file from `~/.stormpath` directory**.
+4. `apiKey.properties` file from application directory**.
```

Comment:
What do you mean "application directory" ?  This appears to be very environment specific and shouldn't be part of the spec.